mne/io/meas_info.py

```diff
@@ -1722,6 +1722,7 @@ def _merge_info(infos, force_update_to_first=False, verbose=None):
     for k in other_fields:
         info[k] = _merge_info_values(infos, k)
 
+    info['meas_date'] = infos[0]['meas_date']
```

Comment:
If you are going to do this, then `meas_date` can be removed from the `other_fields` list as you undo whatever it would have done (I think).

Reply:
thanks to your comment now I understand better the code. What we should do is actually return the right `meas_date` by using `_merge_info_Values(infos, 'meas_date')`.

In other words, if `[_['meas_info'] for _ in infos]` is something like `[(xx,xx), None, None ... None]`, `_merge_info_values` should give you back `(xx, xx)`. 

This should be done here:

https://github.com/mne-tools/mne-python/blob/1e9d40246e5c4273ed968926afa77890a242afb2/mne/io/meas_info.py#L1598-L1601

 However, I have the feeling that is not going down the right path. 'Cos I changed for the following snipped and it does not go over `1/0`:
```py
        elif key == 'meas_date':
            1/0
            if all(x is None for x in values[1:]):
                return values[0]
            else:
                logger.info('Found multiple entries for %s. '
                            'Setting value to `None`' % key)
                return None
```

what I propose is to remove it from `other_fields` for now and inspect the bug in a different PR.

Reply:
fine by me

Reply:
ticket #5517